train_tokenizer.py

```diff
@@ -58,6 +61,9 @@ class ModelArguments:
     min_frequency: int = field(
         default=2,
     )
+    additinal_special_tokens: Optional[List[str]] = field(
```

Comment:
```suggestion
    additional_special_tokens: Optional[List[str]] = field(
```
오타가 있는 것 같은데 확인 부탁드립니다